virtualizarr/tests/test_xarray.py

```diff
@@ -384,6 +395,23 @@ def test_explicit_filetype(self, netcdf4_file):
         with pytest.raises(NotImplementedError):
             open_virtual_dataset(netcdf4_file, filetype="grib")
 
+    def test_group_kwarg(self, hdf5_groups_file):
+        with pytest.raises(ValueError):
+            open_virtual_dataset(
+                hdf5_groups_file
+            )  # ValueError: Multiple HDF Groups found
+        with pytest.raises(ValueError):  # group not found
+            open_virtual_dataset(hdf5_groups_file, group="group")
+
+        vars_to_load = ["air", "time"]
+        vds = open_virtual_dataset(
+            hdf5_groups_file, group="test/group", loadable_variables=vars_to_load
+        )
+        full_ds = xr.open_dataset(hdf5_groups_file, group="test/group")
+        for name in full_ds.variables:
+            if name in vars_to_load:
+                xrt.assert_identical(vds.variables[name], full_ds.variables[name])
```

Comment:
I think I just copied from above https://github.com/zarr-developers/VirtualiZarr/blob/5d0851976778ebaf52eb6df57970b0706a892d98/virtualizarr/tests/test_xarray.py#L398-L400

Reply:
Checking the Virtual dataset against the full dataset raises a blank `AssertionError`. Does something like `virtualizarr.testing.assert_matches(vds, full_ds)` already exist that I've overlooked?

Reply:
I added a fix (in main two commits ago) with https://github.com/zarr-developers/VirtualiZarr/pull/188 that should allow `xr.testing.assert_identical` to work. Let me know if it's still not working

Reply:
@scottyhq if we can use `xr.testing.assert_identical` here we should

Reply:
I don't think #188 resolved this. Added some comments to https://github.com/zarr-developers/VirtualiZarr/issues/161 to explain

Reply:
OK, thanks for the clarifications in the issue above @TomNicholas. I was getting really confused I think because of this conversation here. I thought somehow `xr.testing.assert_identical(vds, full_ds)` was supposed to work rather than `xr.testing.assert_identical(vds1, vds2)` which makes much more sense. In any case, I'll try to do a final clean up today and hopefully merge this.

Reply:
No worries! This is confusing - perhaps I should add a section in the docs explaining what it means to test equality between two `ManifestArrays`...